src/Compilers/Core/Portable/Operations/ControlFlowBranchSemantics.cs

```diff
@@ -1,19 +1,57 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using Microsoft.CodeAnalysis.Operations;
+
 namespace Microsoft.CodeAnalysis.FlowAnalysis
 {
     /// <summary>
-    /// PROTOTYPE(dataflow): Add documentation
+    /// Semantics associated with a <see cref="ControlFlowBranch"/>.
     /// </summary>
     public enum ControlFlowBranchSemantics
     {
+        /// <summary>
+        /// Represents a <see cref="ControlFlowBranch"/> with no associated semantics.
```

Comment:
>Represents a <see cref="ControlFlowBranch"/> with no associated semantics. [](start = 12, length = 74)

I found this surprising. What's the difference between none and regular?

Reply:
I did not find use of None anywhere in the test baselines. @AlekseyTs, is this just the default value?